yb-voyager/cmd/exportData.go

```diff
@@ -178,11 +179,12 @@ func getFinalTableColumnList() ([]*sqlname.SourceName, map[*sqlname.SourceName][
 	var tableList []*sqlname.SourceName
 	// store table list after filtering unsupported or unnecessary tables
 	var finalTableList, skippedTableList []*sqlname.SourceName
-	excludeTableList := extractTableListFromString(source.ExcludeTableList)
+	fullTableList := source.DB().GetAllTableNames()
+	excludeTableList := extractTableListFromString(fullTableList, source.ExcludeTableList, "exclude")
 	if source.TableList != "" {
-		tableList = extractTableListFromString(source.TableList)
+		tableList = extractTableListFromString(fullTableList, source.TableList, "include")
```

Comment:
@priyanshi-yb can you update the table-list (and exclude-table-list) argument help messages to include this pattern support?

Reply:
yes, will do